internal/webrisk_proto/webrisk.proto

```diff
@@ -100,25 +102,30 @@ message ComputeThreatListDiffResponse {
 // Request to check URI entries against threatLists.
 message SearchUrisRequest {
   // The URI to be checked for matches.
-  string uri = 1;
+  repeated string uris = 1;
 
   // Required. The ThreatLists to search in.
   repeated ThreatType threat_types = 2;
+
+  bool local = 3;
 }
 
 message SearchUrisResponse {
   // Contains threat information on a matching uri.
   message ThreatUri {
+    // The pattern for the URI
```

Comment:
Inserting at the front is breaking backwards compatibility.

Reply:
Read comment above for backwards compatibility